release-notes/opensearch-anomaly-detection-dashboards.release-notes-3.0.0.0-beta1.md

```diff
@@ -0,0 +1,19 @@
+## Version 3.0.0.0-beta1 Release Notes
+
+Compatible with OpenSearch Dashboards 3.0.0.0-beta1
+
+### Enhancements
+- Implmentation of contextual launch ([#1005](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/1005))
```

Comment:
I would say this is a feature, instead of an enhancement

Reply:
Sure will update it to feature